{"timestamp": "2025-01-23-17-13-36", "message": "Module error while trying to update local heating control config: failed to set up Google Sheets API: [Errno 2] No such file or directory: '/media/pi/program_stick/kazanfutes/config/secrets_and_env/sheets-access-key.json' (FileNotFoundError).", "severity": 2, "origin": "schedule_and_config_updater.py/update_local_heating_control_config:88 --> project.py/download_google_sheet_to_2D_array:1596", "origin_timestamp": "2025-01-23-17-05-44", "registration_timestamp": "2025-01-23-17-05-44"}
